package.json

```diff
@@ -25,7 +25,7 @@
     "compile": "cross-env NODE_ENV=production webpack --config ./config/webpack.prod.config.js --progress",
     "start": "webpack-dev-server --config ./config/webpack.dev.config.js",
     "start:watch": "npm run clean && webpack --config ./config/webpack.watch.config.js",
-    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\"",
+    "test": "cross-env NODE_ENV=test mocha-webpack \"./src/**/*.spec.js\" \"./test/**/*.js\"",
```

Comment:
I'm not 100% sure, and also very much ok with changing it, however can it be those tests under `./test/**/*.js` are executed by `test:karma`?

Reply:
They are, yes, and I’m happy to undo this, but I found it odd that the test I added weren’t run during local development when just running `npm test` … so yeah, no strong feelings about this :)